### ISSUE TITLE
5- Adds

### ISSUE THREAD

**Issue by samirm00** [samirm00/duck-duck-clone#6]
- [x] Create a branch `adds`
- [x] Create a style.css file 
- [x] Add the code to html file
- [x] Push the branch `adds` to GitHub
- [x] Make a pull request
- [x] Review by the team member 
- [x] Merge the pull request

**Comment by KrystynaMil** [comment]
I did a mistake and made all the changes on the master branch for this issue . But its done and hopefully you will like it ;)

**Comment by samirm00** [comment]
Thank you so much , we have a conflict , but don't worry about it , I will take all the code and place in one file .thank you so much .

**Comment by samirm00** [comment]
@KrystynaMil  did you do the Adds section ??? I have done it too !

**Comment by KrystynaMil** [comment]
Hi, @samirm00 thank you!  Conflict is not a problem . Yes, yesterday I moved all the files in one index.html and style.css(otherwise our page is not working ).Adds section I did and you as well it's good because we are learning. So today in our zoom meeting at 15h we can discuss all the details again :) @Houcinos @MarieBelyse @samirm00  We did a great job and our duckduckClone is almost ready , the rest we will fix all together.
